airflow/_vendor/connexion/decorators/parameter.py

```diff
@@ -6,7 +6,7 @@
 import inflection
 
 from ..http_facts import FORM_CONTENT_TYPES
-from ..lifecycle import ConnexionRequest  # NOQA
+from ..lifecycle import ConnexionRequest
```

Comment:
Generally I'd say we shouldn't check/change code under `airflow/_vendor/` -- so we should exclude this from the Yesqa check if it's complaining and not otherwise touch these files

Reply:
Good point. fixing it